supervisor/conf/config-1.yml

```diff
@@ -0,0 +1,85 @@
+# Connection details for the OpAMP server.
+opampServer:
+    endpoint: https://localhost:1729
+    tls:
+        # Client-side TLS configuration to connect to the OpAMP server.
+
+# Agent specifies the agent that is being managed by this supervisor. Concretely,
+# the `agent' configuration dictates how to start it and where the configuration
+# file resides. This basic information is necessary for the supervisor to
+# restart the agent on a remote configuration change.
+# `watchdog' and `systemd' config elements are mutually exclusive. So, only
+# one of the two must be specified below.
+agent:
+    # Type specifies the type of the agent that is being supervised. Internally,
+    # the `type' can be used to auto-discover several attributes if the
+    # supervisor is tailor-made for that agent thereby relieving the user from
+    # having to specify most, if not all, of the configuration.
+    type: io.opentelemetry.collector
+    # Fully-qualified path to the agent executable required for two reasons:
+    # 1. If the supervisor is functioning as a watchdog, then the executable
+    #    is used to start/stop/restart the agent on configuration changes, etc.
+    # 2. If the supervisor is performing the upgrade, the executable indicates
+    #    the file and path in the filesystem that must be replaced with the newer
+    #    version of the executable.
+    executable: /opt/opentelemetry-collector/bin/otel
+    #
+    management:
+        # Dictates how to check if the agent is healthy.
+        healthCheck:
+            url: http://localhost:13133/
+            initialDelay: 5s
+            period: 5s
+            timeout: 20s
+        ownTelemetry:
+            url: http://localhost:8888/
+            initialDelay: 5s
+            period: 5s
```

Comment:
What does this do?

Reply:
My thinking was to allow the vendor to specify how frequently the telemetry must be scraped by the supervisor. That's what the `period` specifies.

Reply:
However, this is only metrics rather than the full telemetry? 

Also, I think the idea on the own telemetry is that agent can be given the configuration where it needs to be pushed and take care of that itself (however, I think in certain cases it could be also done by supervisor)

Reply:
This implies Supervisor scrapes the Agent for metrics. I would prefer not to do this, otherwise the Supervisor will slowly become a Collector on its own :-)

For Otel Collector I think we should go with a different approach: Supervisor to instruct the Collector where to send the metrics. This should be possible to do by configuring a Collector pipeline. This approach of course won't work for a generic Supervisor. I think it is fine to omit it as a feature in the generic Supervisor initially. We can discuss and decide if we want to make the Supervisor more complicated and be responsible for scraping in the future.